instructions/steps.md

```diff
@@ -384,9 +357,8 @@ Copy over these naive signatures.
 `function` is mostly what you expect.
 However, `function`s in Dafny are more restrictive
 than you are probably used to.
-They are syntactic sugar for expressions.
 They are more like mathematical functions: 100% deterministic.
-The can't mutate, use loops, or create objects on the heap.
+The can't mutate, use loops, or create objects.
```

Comment:
Hum, I'll leave it, but we do create `datatype` in functions.
And the distention may come up.
Not blocking